.taplo.toml

```diff
@@ -0,0 +1,11 @@
+include = ["Cargo.toml", "**/Cargo.toml", "examples/**/Cargo.toml"]
+
+[formatting]
+reorder_keys = false
+
+[[rule]]
+include = ["Cargo.toml", "**/Cargo.toml", "examples/**/Cargo.toml"]
```

Comment:
Is it necessary to duplicate the file listing here?  Surely without this line it will apply to all files anyway.